graph-builder/tests/slo/mod.rs

```diff
@@ -0,0 +1,93 @@
+use failure::Fallible;
+use pretty_assertions::assert_eq;
+use reqwest::header::{HeaderValue, AUTHORIZATION};
+use serde::Deserialize;
+use std::env;
+use test_case::test_case;
+use url::Url;
+
+#[derive(Deserialize)]
+struct PromMetric {
+    value: (f64, String),
+}
+
+#[derive(Deserialize)]
+struct PromData {
+    result: Vec<PromMetric>,
+}
+
+#[derive(Deserialize)]
+struct PromResponse {
+    status: String,
+    data: PromData,
+}
+
+fn query_prom(url: String, token: String) -> Fallible<String> {
+    let mut runtime = commons::testing::init_runtime().unwrap();
```

Comment:
I suggest moving to runtime variable to the test itself, and passing in a `&mut` of it to this function, or even better, make this an `async fn` and return a future.